04/mobile.css

```diff
@@ -0,0 +1,167 @@
+ 
+    @font-face {
+        font-family: 'Source Sans PRO';
+        src: url(source-sans-pro/SourceSansPro-Bold.otf) format('opentype');
+    }
+    
+    @media (max-width: 2000px){
+    body{
+        max-width: 2000px;
+        background-color: rgb(224, 226, 227);
+      
+    }
+
+    label{
+        place-self: center;
+        width: 50px;
+        height: 35px;
+        border: 0;
+        border-top: 5px solid rgb(11, 10, 10);
+        border-bottom: 3px;
+        background-color: transparent;
+        position: relative;
+    }
+    
+    label::before, label::after{
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        border-top: 5px solid  rgb(11, 10, 10);
+        transform: translateY(10px);
+    }
+    
+    label::after{
+        transform: translateY(25px);
+    }
+
+    nav {
+        background-color: rgb(201, 174, 174);
+        display: flex;
+        list-style: none;
+        padding: 0;
+        margin-top: 20px;
+        margin-left: 10px;
+        max-width: 50%;  
+    }
+    
+    nav ul {
+        list-style: none;
+        padding: 0; 
+        display:none;
+        text-align: center;
+        font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode'
+        
+    }
+    
+    nav ul li {
```

Comment:
Dużo łatwiej jest stylować przy pomocy klas. Ten selektor oznacza, że te cechy mogą być przypisane to np takiego znacznika, co mocno utrudnia stylowanie:
```
<nav> 
  <div>
     <ul>
         <li>
             <ul>
                   <li> <!-- ten też będzie miał te style -->
```